unicodetools/org/unicode/text/tools/RenameFiles.java

```diff
@@ -22,7 +22,7 @@
 public class RenameFiles {
 
     // First set the source accordingly.
-    private static final Choice choice = Choice.emojione;
+    private static final Choice choice = Choice.android;
```

Comment:
This is just a an internal utility to correct the file names that vendors supply.